website_sale_wishlist/static/src/css/website_sale_wishlist.css

```diff
@@ -0,0 +1,3 @@
+img.oe-image-wishlist {
```

Comment:
Hi @lasley `oe` make me think 'Odoo ERP', then no needs to deprecate, isn't it ? Joke, yes